server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -514,6 +515,8 @@
     private static final ConfigKey<Boolean> AllowDeployVmIfGivenHostFails = new ConfigKey<Boolean>("Advanced", Boolean.class, "allow.deploy.vm.if.deploy.on.given.host.fails", "false",
             "allow vm to deploy on different host if vm fails to deploy on the given host ", true);
 
+    private static final ConfigKey<Boolean> EnableAdditionalVmConfig = new ConfigKey<>("Advanced", Boolean.class, "enable.additional.vm.configuration",
```

Comment:
Can we change this setting from 'enable.additional.vm.configuration' to 'additional.vm.configuration.allow' or 'additional.vm.configuration.enable'?